src/app/pages/fund/components/fiat-provider-item.tsx

```diff
@@ -4,15 +4,25 @@ import { FastCheckoutBadge } from './fast-checkout-badge';
 import { ZeroPercentFeesBadge } from './zero-percent-fees-badge';
 import { FundAccountTile } from './fund-account-tile';
 
-const availableInsideUnitedStatesDescription = 'Available both inside and outside of the US';
+const availableInsideUnitedStatesDescription = 'Available only inside of the US';
+const availableInsideAndOutsideUnitedStatesDescription =
+  'Available both inside and outside of the US';
```

Comment:
I won't initiate this copy discussion again, but smh 😭 🇺🇸

Reply:
🦅